WordPress/Classes/ViewRelated/Me/MeViewController.swift

```diff
@@ -414,6 +414,26 @@ class MeViewController: UITableViewController, UIViewControllerRestoration {
         headerView.onGravatarPress = { [weak self] in
             self?.presentGravatarPicker()
         }
+        headerView.onDroppedImage = { [weak self] image in
+            let imageCropViewController = ImageCropViewController.init(image: image!)
```

Comment:
I'm wondering if there is a way to avoid `image` being treated as an optional.  Would it make sense for it to be declared as non null in the objc header?

Reply:
I think the way to handle this ideally would be to add nullability annotations to `MeHeaderView.h`, for example:

`typedef void (^MeHeaderViewDropCallback)(UIImage * _Nonnull image);`